doc/tutorials/run_error_correction.py

```diff
@@ -0,0 +1,152 @@
+RHG_code = SurfaceCode(distance=distance, ec=ec, boundaries=boundaries)
+RHG_code.draw()
+
+
+######################################################################
+# By default, the primal nodes are drawn black and dual nodes as grey.
+# 
+
+
+######################################################################
+# Next, we have to add some noise to our ideal qubits. For this tutorial,
+# we opt for a simple Pauli noise model, where each qubit has some
+# independent probability of a phase (Pauli :math:`Z`) error:
+# 
+
+p_Z = 0.3
+IidNoise(RHG_code, p_Z).apply_noise()
+
+
+######################################################################
+# At this point, we seek help from the decoder to correct our noisy
+# lattice. The two decoders currently available in FlamingPy are
+# *minimum-weight perfect matching* (''MWPM'') and *Union-Find* (''UF'').
+# Although MWPM is slower, it is standard and well-performing; let us
+# select it:
+# 
+
+decoder = {"outer": "MWPM"}
+
+
+######################################################################
+# The option ''outer'' here indicates that this is a qubit-level decoder
+# taking in bit values. One may also specify an ''inner'' decoder if the
+# qubits are, for example, GKP states, which are capable of performing an
+# additional round of error correction themselves. We will go over this in
+# more detail in another tutorial.
+# 
+
+
+######################################################################
+# Finally, we can run the ''correct'' function, which decodes and recovers
+# the information. We can illustrate the full decoding procedure, for
+# which we first specify some drawing options:
+# 
+
+dw = {
+    "show_nodes": False,
+    "color_nodes": 'k',
+    "show_recovery": True,
+    "label_stabilizers": True,
+    "label_boundary": True,
+    "label_edges": False,
+    "label": None,
+    "legend": True,
+    "show_title": True,
+    "show_axes": True,
+}
+
+c = dec.correct(
+    code=RHG_code,
+    decoder=decoder,
+    draw=True,
+    drawing_opts=dw,
+)
+print(f"Success: {c}")
+
+
+######################################################################
+# Here's what we have drawn:
+# 
+# -  The *stabilizer graph*, a representation of the codes with
+#    information about the errors that have occurred.
+# -  The *matching graph*, the processed stabilizer graph that is fed into
+#    the decoder.
+# -  The *syndrome and recovery* plot, where we visualize where errors
+#    have ocurred (the red voxels) and the *matching* (lines connecting
+#    the voxels) that shows where the bit values must be flipped to
+#    recover the information.
+# 
+# You can go back and forth between these plots by comparing the indices
+# of the decoding graphs and those on the voxel plot. You may also display
+# the RHG lattice by setting ''show_nodes'' to True.
+# 
+# More information about all these steps will come in future tutorials;
+# for now, you may visit our :ref:``quantum error correction``\ <
```

Comment:
Is there a label for this one?
```suggestion
# for now, you may visit our :ref:`quantum error correction`
```

Reply:
Actually it's our error correction blurb here: https://flamingpy.readthedocs.io/en/qec-blurb/quantum_error_correction.html . I was trying to link to it via sphinx labels rather than an explicit hyperlink